experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -306,4 +327,63 @@ export class NodeSDK {
         .then(() => {})
     );
   }
+
+  private configureLoggerProviderFromEnv(
+    envWithoutDefaults: ENVIRONMENT
+  ): void {
+    const logExportersList = envWithoutDefaults.OTEL_LOGS_EXPORTER;
+    if (!logExportersList) {
+      diag.warn(`No log exporters specified. Logs will not be exported.`);
```

Comment:
I would not consider this a warning. Having the env var not set is the default.

```suggestion
      diag.info(`No log exporters specified. Logs will not be exported.`);
```